objects/src/notes/mod.rs

```diff
@@ -47,15 +46,15 @@ pub const NOTE_LEAF_DEPTH: u8 = NOTE_TREE_DEPTH + 1;
 /// Auxiliary data which is used to verify authenticity and signal additional information:
 /// - A metadata object which contains information about the sender, the tag and the number of
 ///   assets in the note.
-/// - An origin which provides information about the origin of the note.
+/// - A proof which provides the data required to authenticate the note.
```

Comment:
minor nit: maybe we should clarify that this authentication is against note root of a block (rather than the full authentication info starting from chain MMR)?